configs/test.js

```diff
@@ -0,0 +1,12 @@
+const configs = require('./dev');
+
+module.exports = {
+  ...configs,
+  db: {
+    ...configs.db,
+    connection: {
+      ...configs.db.connection,
+      database: 'courses_service_test'
```

Comment:
since the dev database is "courses", I would prefer to use "courses_test". But it is a minor comment, its the same

Reply:
or maybe better "courses_service" and "courses_service_test"

Reply:
done! courses database was changed to courses_service